highfidelity/hifi
# Hitting Enter Key in Password Field To Login

MS [#14261](https://highfidelity.fogbugz.com/f/cases/14261/Clicking-enter-while-focus-is-in-password-field-should-attempt-login)

- adding enter/return key event to password field to allow for login

**TEST PLAN**
For signing in:

- Launch Interface
- If you are logged in, log out via `File > Logout <user>`
- Go to `File > Login/Sign up`
- Type in your username/email
- Type in password
- While still in password text box, hit `Enter`
- You should either see an error at the top of the dialog or see that the sign in was successful

For signing up:

- Launch Interface
- If you are logged in, log out via `File > Logout <user>`
- Go to `File > Login/Sign up`
- Click the `Sign Up` button
- Type in some email
- Type in some username
- Type in some password
- While still in password text box, hit `Enter`
- You should now either see an error at the top of the dialog or see that the sign up was successful
